docs/articles/documentation/reference/test-api/testcontroller/clearupload.md

```diff
@@ -5,10 +5,10 @@ permalink: /documentation/reference/test-api/testcontroller/clearupload.html
 ---
 # t.clearUpload Method
 
-Removes all file paths from the specified file upload input.
+Removes all file paths from the specified file upload input. Can be chained with other `TestController` methods.
 
 ```text
-t.clearUpload( selector )
+t.clearUpload(selector) → this
```

Comment:
Actually, the return value here is a union type of `this | Promise`. You can call chainable methods as well as using `await` or `Promise.then` to start the chain execution.

Reply:
I get that TestControllerPromise extends TestController and Promise.

Should we do

> t.clearUpload(selector) → this | Promise

or rather

> t.clearUpload(selector) → this | Promise<any>

Reply:
@AndreyBelym is `this | Promise<any>` a common notion? Because I wouldn't understand what this means.